src/components/Graph.react.js

```diff
@@ -1,5 +1,6 @@
 import React, {Component, PropTypes} from 'react';
 import {contains, filter, has, type} from 'ramda';
+import {omit} from 'underscore'
```

Comment:
We use `ramda` instead of underscore. Check out their docs here: http://ramdajs.com